ftdetect/terramate.vim

```diff
@@ -0,0 +1,3 @@
+" vint: -ProhibitAutocmdWithNoGroup
+silent! autocmd! filetypedetect BufRead,BufNewFile *.tm.hcl
+autocmd BufRead,BufNewFile *.tm,*.tm.hcl set filetype=terramate
```

Comment:
I have no idea what these lines are doing but the first line has only `*.tm.hcl` but second line has both `*.tm` and `*.tm.hcl`. Why?

Reply:
Fixed